src/clients/aws_cost_usage_client.py

```diff
@@ -0,0 +1,56 @@
+from logging import getLogger
+from typing import Dict, Any
+from datetime import date
+import math
+
+from botocore.client import BaseClient
+from src.data.aws_scanner_exceptions import CostUsageException
+
+
+class AwsCostUsageClient:
```

Comment:
It would be great if this client was named after the AWS service that it interacts with, i.e. `CostExplorer` (if you make this change, related functions in client factory will need to be renamed too).